core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -974,6 +967,34 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
         };
     }
 
+    /**
+     * Reload the URL content map page
+     *
+     * @private
+     * @param {string} inodeOrIdentifier
+     * @memberof EditEmaEditorComponent
+     */
+    private reloadURLContentMapPage(inodeOrIdentifier: string): void {
+        this.dotContentletService
+            .getContentletByInode(inodeOrIdentifier)
+            .pipe(tap(() => this.store.updateEditorState(EDITOR_STATE.LOADING)))
+            .subscribe(({ URL_MAP_FOR_CONTENT }) => {
```

Comment:
If this fails, the Editor state is still in loading?

Reply:
Nice catch, Kevin!